index.d.ts

```diff
@@ -0,0 +1,55 @@
+interface IWickedElementsComponent<A extends Array<string>> {
```

Comment:
any particular reason to make it an extends of an Array ?

Reply:
I think my answer to your earlier question also answers this one, right ? :slightly_smiling_face: